en/Appendices/appendixC/software-and-development-troubleshooting.md

```diff
@@ -0,0 +1,24 @@
+<center><h1>Software and Development Troubleshooting</h1></center>
+
```

Comment:
Did you look at the preview of this?

Reply:
Yes, I have looked at a preview of the file, and it appears as intended. If you click "Display the rich diff" it displays the markdown file without the .png file included. If you click the three dots to the right of that option, and select "View file" from the drop-down, the markdown file will display as intended, with the .png file included.